### PR TITLE
Add the configure using a property file step. Fix bug where you could…

### PR DESCRIPTION
**Pull request by steven1046**
… save the editor and have it complete a future instruction

This pull request will deliver the config properties step but the tabbed editors will come after that has been implemented.

### CODE REVIEW

File: json-guides/microprofile-config.json [hunk 1]
```diff
@@ -15,6 +15,35 @@
             ]
         },
         {
+            "name": "ConfigurePropsFile",
+            "title": "Configure using a Properties File",
+            "description": [
+                "In Websphere Liberty, you can provide a properties file as part of your Liberty environment configuration. This properties file lets you specify configuration for overriding injected values.",
```
Comment (lavenal) [inline json-guides/microprofile-config.json]: The properties file could be provided in different way. The easy way is to package it as part of the application. If we want to provide it as part of the liberty environment, it has to be packaged as a shared lib.

File: json-guides/microprofile-config.json [hunk 2]
```diff
@@ -15,6 +15,35 @@
...
+            ],
+            "instruction": [
+              "Add the <code>download-url=ftp://music.com/canada/download</code> on line 1 or click <action title='Configure download-url in microprofile-config.properties' onclick=\"microprofileConfigCallBack.addPropToConfigProps(event)\"><b>download-url=ftp://music.com/canada/download</b></action>. Then, click <action title='Save' onclick=\"microprofileConfigCallBack.saveServerEnvButton(event)\"><b>Save</b></action> on the editor menu pane.",
+              "Click <action title='Refresh' onclick=\"microprofileConfigCallBack.serverEnvRefreshBrowserButton(event)\"><b>Refresh</b></action> in the following browser to see that the download-url in the microprofile-config.properties file is used.<br/>"
```
Comment (lavenal) [inline json-guides/microprofile-config.json]: I've changed the action to use Run.

File: json-guides/microprofile-config.json [hunk 3]
```diff
@@ -15,6 +15,35 @@
...
+            ],
+            "content":[
+              {
+                "displayType":"fileEditor",
+                "fileName": "microprofile-config.properties",
+                "preload": [
+                ],
+                "save": true,
```
Comment (lavenal) [inline json-guides/microprofile-config.json]: If using Run, need to change save to false.

**Comment by steven1046** [comment]
@lavenal I updated the code to use 'Run'.

File: json-guides/microprofile-config.json [hunk 4]
```diff
@@ -15,6 +15,35 @@
             ]
         },
         {
+            "name": "ConfigurePropsFile",
+            "title": "Configure using a Properties File",
+            "description": [
+                "In Websphere Liberty, you can provide a properties file as part of your Liberty environment configuration. This properties file lets you specify configuration for overriding injected values.",
+                "",
+                "Add the download-url property to the microprofile-config.properties file to override the injected value of the download-url. The properties file contains settings with a default ordinal of 100 which overrides the injected properties with the same name. If a property appears in more than one source, then the property value from the source with the highest ordinal takes precedence. The download-url property in the microprofile-config.properties will be used because the injected default value is only used when no other config sources are provided."
```
Comment (lavenal) [inline json-guides/microprofile-config.json]: Sorry, forget to say that the file has to be in the META-INF directory. Change it to 

Add the download-url property to the <code>META-INF/microprofile-config.properties</code> file ...

**Pull request merged**